src/note.rs

```diff
@@ -345,6 +354,32 @@ impl Note {
             ..self
         }
     }
+
+    /// Update the rho value of the issuance note (see
+    /// [ZIP-227: Issuance of Zcash Shielded Assets][zip227]).
+    ///
+    /// [zip227]: https://zips.z.cash/zip-0227
+    pub(crate) fn update_rho_for_issuance_note(
+        &mut self,
+        nullifier: Nullifier,
```

Comment:
Let's call it `first_nullifier` to indicate that the required value is the first nullifier of the transfer bundle.
Also, please update the documentation to support it.

Reply:
Done